src/components/Tabs/Tabs.jsx

```diff
@@ -0,0 +1,634 @@
+import React, { useState, useEffect } from 'react'
+import styled, { css, up, down } from '@xstyled/styled-components'
+import { variant } from '@xstyled/system'
+import PropTypes from 'prop-types'
+import { outerWidth, debounce } from '../../utils'
+
+import { Typography } from '../'
+import { Icon } from '../Iconography'
+import Flex from '../Grid/Flex'
```

Comment:
Aqui pode importar assim pra manter o padrão

```suggestion
import { Flex } from '../Grid'
```